content/en/docs/tasks/observability/distributed-tracing/opentelemetry/http_test.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154,SC2155,SC2034,SC2016
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source "tests/util/samples.sh"
+source "tests/util/addons.sh"
+
```

Comment:
# @setup profile=demo

we should use demo profile, otherwise you need to patch meshconfig manually.

Reply:
Hum but we do need to patch the meshconfig manually to configure the HTTP exporter. I did the same as done for the Skywalking test/page, as they also need to modify to add their extension to meshconfig.

Reply:
checkout this https://github.com/istio/istio.io/tree/abf5268f8aff3a5dd1b78c384873d414e6fb1468/content/en/docs/tasks/observability/logs/otel-provider

may need upgrade package istio/istio to latest

Reply:
<img width="572" alt="image" src="https://github.com/istio/istio.io/assets/4354057/878d7c5c-d973-4217-a486-b5409c253326">

wrong profile will waste time to run test on each profile

Reply:
Ah OK. I changed to default profile for the gRPC one at least. I hope that helps. 

> checkout this [`abf5268`/content/en/docs/tasks/observability/logs/otel-provider](https://github.com/istio/istio.io/tree/abf5268f8aff3a5dd1b78c384873d414e6fb1468/content/en/docs/tasks/observability/logs/otel-provider)
> 
> may need upgrade package istio/istio to latest

Not sure what you mean with that? Do I need to update something? 🤔 I'm already based on laster master

Reply:
Updated with the commit in Istio where the HTTP exporter was added. (2 behind latest master)